app/src/main/java/org/oppia/app/player/state/ImageRegionSelectionInteractionView.kt

```diff
@@ -0,0 +1,98 @@
+package org.oppia.app.player.state
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.FrameLayout
+import androidx.appcompat.widget.AppCompatImageView
+import androidx.databinding.BindingAdapter
+import androidx.fragment.app.FragmentManager
+import org.oppia.app.fragment.InjectableFragment
+import org.oppia.app.model.ImageWithRegions
+import org.oppia.app.utility.ClickableAreasImage
+import org.oppia.app.utility.OnClickableAreaClickedListener
+import org.oppia.util.accessibility.CustomAccessibilityManager
+import org.oppia.util.gcsresource.DefaultResourceBucketName
+import org.oppia.util.parser.ExplorationHtmlParserEntityType
+import javax.inject.Inject
+
+/**
+ * A custom [AppCompatImageView] with a list of [LabeledRegion] to work with
+ * [ClickableAreasImage].
+ */
+class ImageRegionSelectionInteractionView @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyleAttr: Int = 0
+) : AppCompatImageView(context, attrs, defStyleAttr) {
+
+  private var isAccessibilityEnabled: Boolean = false
```

Comment:
Should this be a ``lateinit``, instead? We technically don't know the answer to this until ``onAttachedToWindow``, so we probably shouldn't default it.

Reply:
this is primitive delegate can't be used lateinit

Reply:
this can't be done as this is being a primitive data type

Reply:
Ah, got it. Good point.